Dockerfile

```diff
@@ -103,21 +138,13 @@ RUN export HOME_OVERRRIDE=/tmp/home && mkdir -pv /tmp/home/pfiles && \
     cd /heasoft-heasp && \
     hmake install
 
-
-
-#RUN export HOME_OVERRRIDE=/tmp/home && mkdir -pv /tmp/home/pfiles && \
-#    source /init.sh && \
-#    python -c 'import xspec; print(xspec.__file__)' && \
-#    pip install numpy scipy ipython jupyter matplotlib pandas astropy==2.0.11
-
-RUN export HOME_OVERRRIDE=/tmp/home && mkdir -pv /tmp/home/pfiles && \
-    source /init.sh && \
-    pip install numpy scipy ipython jupyter matplotlib pandas astropy==2.0.11
-
+#pip install -r https://raw.githubusercontent.com/volodymyrss/data-analysis/py3/requirements.txt && \
+#Removed healpy version from requirements
+ADD data-analysis-requirements.txt /tmp/data-analysis-requirements.txt
```

Comment:
While this container was made long ago when py3 transition was under way, as of now you can just install the master branch, I suggest to try that.
Also it might and should just work without requirements at this time.